plugins/techdocs-backend/src/plugin.ts

```diff
@@ -64,6 +67,13 @@ export const techdocsPlugin = createBackendPlugin({
       },
     });
 
+    const customPreparers = new Map<RemoteProtocol, PreparerBase>();
+    env.registerExtensionPoint(techdocsPreparerExtensionPoint, {
+      registerPreparer(protocol: RemoteProtocol, preparer: PreparerBase) {
+        customPreparers.set(protocol, preparer);
```

Comment:
Perhaps we can throw an error if it's already registered here? Just to avoid mistakes, since ordering of modules isn't guaranteed.